.circleci/config.yml

```
@@ -244,7 +244,7 @@ jobs:
       - *java_deps
       - run:
           name: Validate views
-          command: PATH="venv/bin:$PATH" script/validate_views
+          command: script/bqetl view validate
```

oop, I got this wrong, should be:
```suggestion
          command: PATH="venv/bin:$PATH" script/bqetl view validate
```